.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
   - "node"
+  - "lts/*"
```

Comment:
Instructs Travis CI to build the project with the latest Long Term Support release of Node.js (currently 6.*) to ensure the project maintains compatibility.